proposals/cloudevents.md

```diff
@@ -0,0 +1,188 @@
+# Proposal: CloudEvents as unified event data format
```

Comment:
general comments: 
- suggesting to add the software application that already supports cloud events.  
- should we mention as well CloudEvents schema registry and service discovery?